service/authorization/authorization_test.go

```diff
@@ -26,14 +31,65 @@ var (
 	mockFqn2                         = fmt.Sprintf("https://%s/attr/%s/value/%s", mockNamespace, mockAttrName, mockAttrValue2)
 )
 
-func mockRetrieveAttributeDefinitions(ctx context.Context, _ *authorization.ResourceAttribute, _ *otdf.SDK) (map[string]*attr.GetAttributeValuesByFqnsResponse_AttributeAndValue, error) {
-	slog.DebugContext(ctx, "Using mocked GetAttributeValuesByFqns: "+getAttributesByValueFqnsResponse.String())
-	return getAttributesByValueFqnsResponse.GetFqnAttributeValues(), nil
-}
+type myAttributesClient struct{}
 
-func mockRetrieveEntitlements(ctx context.Context, _ *authorization.GetEntitlementsRequest, _ *AuthorizationService) (*authorization.GetEntitlementsResponse, error) {
-	slog.DebugContext(ctx, "Using mocked GetEntitlements: "+entitlementsResponse.String())
-	return &entitlementsResponse, nil
+func (m *myAttributesClient) ListAttributes(_ context.Context, _ *attr.ListAttributesRequest, _ ...grpc.CallOption) (*attr.ListAttributesResponse, error) {
```

Comment:
I think you only have to add the methods that are actually called. Are all of these called?

Reply:
jk you were right, thanks @jakedoublev for the fix!